polkadot/xcm/xcm-builder/xcm-payment-runtime-api/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "xcm-payment-runtime-api"
```

Comment:
Since it's its own crate, don't we want to move it out of the `xcm-builder` folder?

Reply:
Yes, we do. When choosing the location, we were guided by the fact that `pallet-transaction-payment-rpc-runtime-api` is located inside another crate, and decided to follow this style.
What do you say if we move it to a higher level (to the `xcm` folder)?